### PR TITLE
solved alphabetic shift

### PR DESCRIPTION
**Pull request by rajeshdh**

### CODE REVIEW

File: js-exercises/alphabetic-shift/alphabeticShift.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const alphabeticShift = input => {
+  if (input && input.length) {
+    return input
+      .split('')
+      .map(char => String.fromCharCode(char.charCodeAt(0) + 1))
```
Comment (tejas-hosamani) [inline js-exercises/alphabetic-shift/alphabeticShift.js:5]: Did you consider what happens when a given string has a letter `z` or `Z`?

File: js-exercises/alphabetic-shift/alphabeticShift.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    expect(alphabeticShift('Floccinaucinihilipilification'))
+      .toEqual('Gmpddjobvdjojijmjqjmjgjdbujpo');
+
+    expect(alphabeticShift('Antidisestablishmentarianism'))
+      .toEqual('Boujejtftubcmjtinfoubsjbojtn');
+
+    expect(alphabeticShift('supercalifragilisticexpialidocious'))
+      .toEqual('tvqfsdbmjgsbhjmjtujdfyqjbmjepdjpvt');
+  });
+});
```
Comment (tejas-hosamani) [inline js-exercises/alphabetic-shift/alphabeticShift.test.js:20]: I think if you had written an extra test, you'd have thought of few other edge cases too :)
Example, input string containing a char `z` or `Z`

**Review by tejas-hosamani — COMMENTED** [review]
Overall, it's readable, understandable :)

File: js-exercises/alphabetic-shift/alphabeticShift.js (hunk 1 above)
Comment (yogigachinmath) [reply js-exercises/alphabetic-shift/alphabeticShift.js:5]: Here It is not required to convert it to an array and use map instead you can use for of loop for string directly.
